.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: node_js
+node_js:
+  - lts/*
+cache:
+  directories:
+  - node_modules
+install:
+  - yarn install
+script:
+  - yarn run prettier
+  - yarn run lint
```

Comment:
It it not required to run prettier and linter by using two separate commands. 
With `eslint` (not `tslint`) it is possible to run `yarn run eslint` and it will automatically run prettier. Please, look at the article attached below (in another comment to the pull-request)